TestVectors/unsigned.json

```diff
@@ -0,0 +1,16 @@
+{
+    "@context": [
+        "https://www.w3.org/ns/credentials/v2",
+        "https://www.w3.org/ns/credentials/examples/v2"
+    ],
+    "id": "urn:uuid:58172aac-d8ba-11ed-83dd-0b3aef56cc33",
+    "type": ["VerifiableCredential", "AlumniCredential"],
+    "name": "Alumni Credential",
+    "description": "An minimum viable example of an Alumni Credential.",
```

Comment:
```suggestion
    "description": "A minimum viable example of an Alumni Credential.",
```

Reply:
Good catch @TallTed I'll fix the base unsigned document to be something like:

```json
{
    "@context": [
        "https://www.w3.org/ns/credentials/v2",
        "https://www.w3.org/ns/credentials/examples/v2"
    ],
    "id": "urn:uuid:58172aac-d8ba-11ed-83dd-0b3aef56cc33",
    "type": ["VerifiableCredential", "AlumniCredential"],
    "name": "Alumni Credential",
    "description": "A minimum viable example of an Alumni Credential.",
    "issuer": "https://vc.example/issuers/5678",
    "validFrom": "2023-01-01T00:00:00Z",
    "credentialSubject": {
        "id": "did:example:abcdefgh",
        "alumniOf": "The School of Examples"
    }
}
```

I say something like this, because I'm still trying to resolve with the JSON-LD experts what should be the `@context` and whether I need two or more different unsigned documents for the test vector generation.

Reply:
This instance of `An minimum` does not appear to have been fixed. All other instances appear to have been fixed.